flytekit/extras/accelerators.py

```diff
@@ -1,3 +1,99 @@
+"""
+Use specific accelerators
+==========================
+
+.. tags:: MachineLearning, Advanced, Hardware
+
+Flyte allows you to specify `gpu` resources for a given task. However, in some cases, you may want to use a different
+accelerator type, such as TPU, specific variations of GPUs, or fractional GPUs. You can configure the Flyte backend to
+use your preferred accelerators, and those who write workflow code can import the `flytekit.extras.accelerators` module
+to specify an accelerator in the task decorator.
+
+
+If you want to use a specific GPU device, you can pass the device name directly to the task decorator, e.g.:
+
+.. code-block::
+
+    @task(
+        limits=Resources(gpu="1"),
+        accelerator="nvidia-tesla-v100",
+    )
+    def my_task() -> None:
+        ...
+
+
+Base Classes
+------------
+These classes can be used to create custom accelerator type constants. For example, you can create a TPU accelerator
+ type as follows:
+
+.. code-block::
+
+    class TPUAccelerator(BaseAccelerator):
```

Comment:
Not supported yet since this requires a different resource name than `nvidia.com/gpu`.